config/paths.json

```diff
@@ -13,6 +13,6 @@
   "packageTest": "__tests__/packageTest",
   "ports": {
     "app": 3000,
-    "test": 8888
+    "test": 8887
```

Comment:
Why does this port number need changing?

Reply:
Doh.  I have conflicts every time I run the tests, I didn't mean to change it ... but if it's OK to change it I won't complain.   8888 is a common port, it might be worth moving away from it but unless that's immediately unanimous I'll revert this change.

Reply:
That's fine. As you say 8887 is probably less likely to conflict so happy to leave it.